### DISCUSSION TITLE
Browsing data

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2433] [Q&A]
Does this browser clear cookies,history and cache?

**Comment by PalmerAL** [comment]
Not automatically, but you can open the history panel (clock icon in the searchbar), and there's a "clear all history" button that will also clear cookies.

**Reply by ghost** [reply]
Thanks!
And when I delete selected browsing items and not the whole history, do I also delete cookies from them?
My apologies from the English language!

**Reply by PalmerAL** [reply]
No, it does not.
